scrypt.go

```diff
@@ -0,0 +1,37 @@
+//go:build !cmd_go_bootstrap
+
+package openssl
+
+// #include "goopenssl.h"
+import "C"
+import (
+	"unsafe"
+)
+
+func Scrypt(password string, salt []byte, N, r, p, maxmem, keylen uint64) ([]byte, error) {
```

Comment:
Nit: use Go  naming conventions

```suggestion
func Scrypt(password string, salt []byte, N, r, p, maxMem, keyLen uint64) ([]byte, error) {
```